### ISSUE TITLE
main-thread-tasks audit does not use toplevel events

### ISSUE THREAD

**Issue by alekseykulikov** [GoogleChrome/lighthouse#7700]
#### Provide the steps to reproduce
1. Run LH on any URL
2. Compare `main-thread-tasks` and `diagnostics`. It's not possible to reproduce diagnostics using data from `main-thread-tasks`.

#### What is the current behavior?

`main-thread-tasks` audit exclude toplevel events:
https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-core/audits/main-thread-tasks.js#L35

`diagnostics` uses only toplevel events:
https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-core/audits/diagnostics.js#L40

#### What is the expected behavior?

It looks like a typo in `main-thread-tasks` and the filter expression should be: 

```js
const results = tasks.filter(task => task.duration > 5 && !task.parent)
```
I can provide a PR if the proposed change is correct.
Or do I have a wrong assumption that `main-thread-tasks` audit should not accompany `diagnostics`?

#### Environment Information
* Lighthouse version: 4.2.0

**Comment by patrickhulce** [comment]
thanks for filing @alekseykulikov this is indeed a typo, I guess this is the problem with adding audits that aren't used for anything :)

the `main-thread-tasks` audit should be `!task.parent`, diagnostics audit is correct

**Comment by alekseykulikov** [comment]
@patrickhulce thank you for fixing it quickly!
The audit is extremely useful, as well as other diagnostics added in v4.2.0.

**Comment by patrickhulce** [comment]
Fixed by #7711
